datadog/__init__.py

```diff
@@ -37,7 +37,9 @@ def initialize(
     api_host=None,  # type: Optional[str]
     statsd_host=None,  # type: Optional[str]
     statsd_port=None,  # type: Optional[int]
+    statsd_disable_aggregating=False,  # type: bool
```

Comment:
I think at least for now we should default this to disabled. Once we do more testing and wider deployments on more workloads we can consider making it the default :)